lib/minitest/reporters/nutrio_reporter.rb

```diff
@@ -0,0 +1,141 @@
+require 'ansi/code'
+
+class NutrioReporter < Minitest::Reporters::BaseReporter
```

Comment:
> I can certainly clean it up

please remove everything that isn't specific to our needs, which i believe is merely the time formatting.